### ISSUE TITLE
Cycling visibility/collapsing content

### ISSUE THREAD

**Issue by TobiasBales** [jsonreeder/vscode-org-mode#70]
First off, thank you for making this extension, it adds one of the two big missing things from emacs (the other being magit)

I am curious, given something like

* My Project
** Tooling
*** TODO do somethings
~here be code,
so that I don't forget~
*** TODO do something else
maybe with some explanation

Is it possible to cycle the visibility/collapsing the code/text blocks? I haven't found anything yet.

If not, would you be interested in adding it/are the contribution guidelines/anything to know before I take a look?

**Comment by ajtoo** [comment]
Hi TobiasBales,

You're welcome, we're glad people like yourself are using it! This extension is still being actively developed and we have visibility/collapsing planned as a future feature (content folding). VS Code handles folding primarily by indentation and VS Code works (based on my understanding) as a "what you see is what you get" editor. That gives us a few things to deal with so it may take a little time. Thanks for your patience and we hope you are pleased when that feature is finally implemented.

TLDR: yes this will be a feature, please be patient :)

**Comment by jsonreeder** [comment]
I miss magit so much! Totally feel you there.

One quick hack is just to add a space before each line that you want to fold. Then run VS Code's folding feature on the header above and it will fold.

I'm hoping we can add a simple feature to add in that space (sort of `org-indent-mode`). Then maybe we'll tackle the real `org-cycle`.

Your contributions would be totally welcome. There are some very basic contribution guidelines in the readme. If you think you might be able to jump into that in the next week or so, send me an email and we can talk details `info@jasonreeder.com`.

**Comment by TobiasBales** [comment]
@jsonreeder thank you, somehow I missed the space indent method in vs code.
Works for me and is good enough for now, thank you so much!

**Comment by jsonreeder** [comment]
Awesome. Don't hesitate to reach out with further ideas or requests.
